__test__/users.test.tsx

```diff
@@ -0,0 +1,174 @@
+import { userHandler } from "../pages/api/users";
```

Comment:
I think we are missing some important tests here, Some off the top of my head:

- Editing a user that does exist
- Editing a user that does not exist 
- 2 users with the same email should not enter the database

Reply:
Also, lets rename the file to .ts instead of .tsx. idk what I was thinking using .tsx

Reply:
Throughout the file, lets use the status codes library instead of hardcoding

Reply:
@anvitaa22 can you do the edit tests?